src/pages/CreateItinerary.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Card, CardBody, Input, Button, Divider } from "@nextui-org/react";
+
+export default function CreateItinerary() {
+    const PlusIcon = () => (
```

Comment:
The `PlusIcon` function isn't needed here, as optimizing your asset management can be done more effectively. By properly organizing your assets in the `assets` directory and importing SVG files as React components, Vite will automatically optimize them for performance. 
This not only streamlines your code but also reduces overhead, making your component more efficient and maintainable. 
Refer this article: [Importing SVG files as React components with Vite](https://dev.to/cassidoo/importing-svg-files-as-react-components-with-vite-l3n).